app-shared/src/androidJvmMain/kotlin/io/github/droidkaigi/confsched/KaigiAppUi.androidJvm.kt

```diff
@@ -183,7 +185,7 @@ actual fun KaigiAppUi() {
                         }
                     },
                     onForwardPressed = {
-                        // TODO Implement the functionality to transition to the next screen.
+                        doForward()
                     },
```

Comment:
I’m actually thinking something like this would be ideal, but making this change would be a separate task:
```kotlin
modifier = Modifier
    .hazeSource(hazeState)
    .mouseNavigationSupportForDesktop(backStack)
```

This approach would help keep the Android/JVM shared code clean.

Reply:
@kitakkun 
I created the task as follows. 🫡 
#587